src/handlers.jl

```diff
@@ -1,68 +1,77 @@
 # --------------------------------------------------------------------
 # plain TXT
 
-function test_reference(file::File{format"TXT"}, actual; render = Diff())
-    _test_reference(render, file, string(actual))
+function test_reference(file::File{format"TXT"}, actual; test = isequal, render = Diff())
+    _test_reference(test, render, file, string(actual))
 end
 
-function test_reference(file::File{format"TXT"}, actual::AbstractArray{<:AbstractString}; render = Diff())
+function test_reference(file::File{format"TXT"}, actual::AbstractArray{<:AbstractString}; test = isequal, render = Diff())
     str = join(actual, '\n')
-    _test_reference(render, file, str)
+    _test_reference(test, render, file, str)
 end
 
 # ---------------------------------
 # Image
 
-function test_reference(file::File, actual::AbstractArray{<:Colorant}; sigma=ones(length(axes(actual))), eps=0.01)
-    _test_reference(BeforeAfterImage(), file, actual) do reference, actual
-        try
-            Images.@test_approx_eq_sigma_eps(reference, actual, sigma, eps)
-            return true
-        catch err
-            if err isa ErrorException
-                return false
-            else
-                rethrow()
-            end
+function default_image_test(reference, actual)
+    try
+        Images.@test_approx_eq_sigma_eps(reference, actual, ones(length(axes(actual))), 0.01)
+        return true
+    catch err
+        if err isa ErrorException
+            return false
+        else
+            rethrow()
         end
     end
 end
 
+function test_reference(file::File, actual::AbstractArray{<:Colorant}; test = default_image_test, render = BeforeAfterImage())
```

Comment:
API changed:

`sigma` and `eps` are deleted here. I believe nobody is using these two keywords, so I didn't make any deprecation here.